pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[build-system]
+requires = ["setuptools>=72", "wheel", "setuptools_scm[toml]>=8"]
+
+[project]
+name = "etos_test_runner"
+dynamic = ["version"]
+description = "ETOS Test Runner"
+authors = [{name = "Tobias Persson", email = "tobias.persson@axis.com"}]
```

Comment:
Do we want to put all the "blame" on @t-persson ? ;)

Reply:
It's kind of unclear what this field describes/conveys?

Reply:
I guess this parameter must be present here with some value. Maintaining the list of all authors would be difficult in each repo's `pyproject.toml`. Sometimes there's also an author list in the documentation. 

If necessary, it probably would be better to generate this list from the Git history.